src/js/player.js

```diff
@@ -3869,7 +3869,8 @@ class Player extends Component {
     // Any mouse movement will be considered user activity
     this.on('mousedown', handleMouseDown);
     this.on('mousemove', handleMouseMove);
-    this.on('mouseup', handleMouseUp);
+    this.on('mouseup', handleMouseUpAndMouseOut);
+    this.on('mouseout', handleMouseUpAndMouseOut);
```

Comment:
I think it would be better to use `mouseleave` so that we don't reset this unnecessarily if we go through other elements in the player (like mousing around in the control bar).
I see it being triggered when leaving the player even with a right click.

Reply:
Yeah, It is a great idea. I have just pushed the changes.